planex/depend.py

```diff
@@ -167,7 +181,9 @@ def main():
                             defines=macros)
             pins[os.path.basename(pin_path)] = spec
 
-    for spec_path in args.specs:
+    links = {pkgname(lnk): lnk for lnk in args.specs if lnk.endswith(".lnk")}
+
+    for spec_path in [spec for spec in args.specs if spec.endswith(".spec")]:
```

Comment:
Several other planex tools and the Makefile rules assume that the file extensions will be lowercase.   rpmbuild will create an SRPM containing foo.SPEC, but mock can't then build it.   Making the Makefile rules case insensitive is also not straightforward, so I think assuming lowercase is the best we can do.